templates/registry/dashboards/dashboard-builder.go.j2

```diff
@@ -50,7 +62,12 @@ func main() {
 			Datasource(prometheusDataSourceRef).
 			WithTarget(
 				prometheus.NewDataqueryBuilder().
-					Expr(`"{"{{metric.metric_name}}"}"`).
+					Expr(`sum({
```

Comment:
Correct me if I'm wrong @lmolkova, but if I understand correctly, Gauges in OpenTelemetry aren't supposed to be summed, only UpDownCounters

Reply:
you're right this doesn't make sense. What'd be the correct aggregation here? averaging?